brew_view/static/src/js/controllers/command_index.js

```diff
@@ -71,16 +71,40 @@ export default function commandIndexController(
     .withBootstrap();
 
   $scope.successCallback = function(response) {
-    // Make sure systems load before moving on with the sorting
```

Comment:
Is this no longer valid?

Reply:
This method feels worth testing.

Reply:
It's not - we no longer care about the rootscope systems list because we're doing a full system list GET (*with* commands) regardless.